database/sql/webhooks.go

```diff
@@ -29,7 +29,7 @@ const (
 
 	sqlDeleteWebhookByUrl = `
 	DELETE FROM webhooks
-	WHERE url = ?
```

Comment:
I can see two methods of placeholdering the params - named and with-`?`. Besides inconsistency, I'm wondering why you've changed the `?` to named parameter (`url`) in `sqlDeleteWebhookByUrl` but in the same time you've left, for instance, the `sqlGetWebhookByUrl` unchanged - which looks quite similar to me.

Reply:
@chris-4chain - answer is simple, the rest endpoints are working properly. And while resolving bugs not doing improvements I try to follow minimal-change approach to make it easier to test as is't required for v1.

Inconsistency was there before:
- for insert methods there were params with :something but for other endpoints it was ?

I can change it everywhere if needed, no problem.

Reply:
If it works - don't change it 😄